gtsfm/frontend/detector_descriptor/detector_descriptor_base.py

```diff
@@ -57,6 +57,8 @@ def create_computation_graph(self, image_graph: Delayed) -> Tuple[Delayed, Delay
         # get delayed object, cannot separate two arguments immediately
         joint_graph = dask.delayed(self.detect_and_describe)(image_graph)
 
+        # TODO: log the number of keypoints from each method (SIFT vs. SuperGlue)
+
```

Comment:
Need this in this PR?

Reply:
Thanks for the review -- I'll put it in an issue instead